pkgs/applications/networking/python-eduvpn-client/default.nix

```diff
@@ -0,0 +1,46 @@
+{ pkgs, stdenv, fetchFromGitHub, pythonPackages, gtk3, libnotify
+, gobjectIntrospection, networkmanager_openvpn, wrapGAppsHook
+, gdk_pixbuf }:
+
+  with pythonPackages;
+
+  buildPythonApplication rec {
+  pname = "eduvpn-client";
+  version = "1.0rc11";
```

Comment:
Why not use a stable version from PyPI? https://pypi.org/project/eduvpn-client/

Reply:
This is what is packaged by the project itself for Debian etc. And it is officially tagged as the latest release candidate ...

Followed the other suggestions. Thanks for reviewing!